docs/pages/versions/unversioned/config/metro.mdx

```diff
@@ -450,6 +450,74 @@ Expo CLI will skip the `./my-module.js` dependency and assume that the developer
 
 Many React libraries shipped the Webpack `/* webpackIgnore: true */` comment to achieve similar behavior. To bridge the gap, we've also added support for Webpack's comment but recommend using the Metro equivalent in your app.
 
+## ES Module Resolution
+
+> This sections applies from SDK 53 on all platforms.
+
+Metro resolves ES Module imports and CommonJS requires with separate resolution strategies.
+
+The classic Node.js module resolution is the strategy that Metro used to apply with some additions to support ES Modules. In this resolution strategy, which matches Node.js versions before v12, Metro resolves modules from `node_modules`, JS files, optionally while omitting extensions, such as `.js`, and uses `package.json` fields such as `main`, `module`, and `react-native`.
+
+In the modern ES Modules resolution strategy, Metro instead resolves modules from `node_modules`, then matches different `package.json` fields, such as `exports`, [a nested map of sub-paths a package exposes](https://nodejs.org/api/packages.html#conditional-exports), and `main`.
+
+Depending on how a package is imported, one of these two resolution strategies will be used. Typically, a file that _imports_ from a Node module, will use the ES Modules resolution strategy, and fall back on regular classic Node.js resolution, but a file that wasn't resolved with ES Modules resolution or has been required with CommonJS resolution, will use the classic resolution strategy.
```

Comment:
the fallback here is not clear. i guess it means the package doesn't have the `package.json:exports`, then metro will fallback to use CJS resolutions (file lookup). would be good to describe what's the fallback is

Reply:
I wish I could :sweat_smile: I'm currently not 100% sure how the fallback works. It might be trying to first use `package.json:exports` with different conditions on a new run, or it might fully fall back to classic resolution.

It's probably the latter, hence the docs saying:
> and fall back on regular classic Node.js resolution

As per the explanation two paragraphs above, that should be accurate-ish